src/frontend/src/common/types/redux.d.ts

```diff
@@ -1,8 +1,20 @@
-type ActionType<T> = (payload: T | null) => {
-  type: string;
-  payload: T | null;
+type ActionType<T> = (payload?: T) => {
+  type: CZGEReduxActions;
+  payload?: T;
 };
 
 enum Pathogen {
   COVID = "covid",
 }
+
+// export action type for use in reducers and middleware
+enum CZGEReduxActions {
```

Comment:
This comment goes for both `CZGEReduxActions` and `ReduxPersistenceTokens` -- 
I personally always find the `.d.ts` files a little... invisible? I dunno, they never come first to mind as places to go look, and because you aren't explicitly importing them elsewhere, I have a tendency to forget about them. Maybe that's on me. For types, I think it's reasonable since types sort of make up the soup that all the code swims in, so it being a bit invisible isn't a big deal. But it just feels like the enums of actions and persistence tokens are more in the "concrete" world of stuff I would _expect_ to import?

I dunno, if you think this is the right place for these, that's fine. But my gut says I would expect `ActionType` to be pulled in automatically from `.d.ts` somewhere, but I would need to explicitly import `CZGEReduxActions` from some `/common/redux` or whatever if I was going to be dispatching or reducing.

Reply:
It's not possible to pull the enums outs without also pulling out ActionType, because the declaration files aren't modules so you can't import anything into them.

However, I put it into a types.ts file which will require importing for consuming modules, as you requested (although, I personally like the magic of not having to import common type, and prefer to just search for them when needed).

Reply:
I think this comes down to me not seeing the redux action type as a _type_, per se. To me, it's a not a TypeScript type indicating the underlying data structure, but rather a constant that just serves as a signal. Now, considering that TS has a whole thing about constant valued types, maybe that's just silly of me. It could simply be that all my experience with redux was in a JS context, so seeing an action type as _also_ being a TS type feels weird to me just because it's new. So yeah, if you prefer it the other way around, you can revert this change in a future PR and I'll just get used to it.